packages/flame/lib/src/components/component.dart

```diff
@@ -73,6 +73,14 @@ class Component {
   /// If two components share the same priority, they will probably be drawn in
   /// the order they were added.
   int get priority => _priority;
+  set priority(int newPriority) {
+    if (parent == null) {
+      _priority = newPriority;
```

Comment:
The only component without a parent is FlameGame, but does it have a notion of priority?

Reply:
Any component can be without a parent as long as it is not mounted

Reply:
Ah, makes sense